ansible/roles_ocp_workloads/ocp4_workload_cyberark_dap/templates/dap_authn_role.j2

```diff
@@ -0,0 +1,18 @@
+# Create cluster role for authentication service access to pods
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dap-authn-role
+rules:
+- apiGroups: [""]
+  resources: ["pods", "serviceaccounts"]
+  verbs: ["get", "list"]
+- apiGroups: ["extensions"]
+  resources: [ "deployments", "replicasets"]
+  verbs: ["get", "list"]
+- apiGroups: ["apps"]
+  resources: [ "deployments", "statefulsets", "replicasets"]
+  verbs: ["get", "list"]
+- apiGroups: [""]
+  resources: ["pods/exec"]
+  verbs: ["create", "get"]
```

Comment:
Please avoid use of the inline JSON style YAML in favor of using normal YAML style everywhere:
```
- apiGroups:
  - ""
  resources:
  - pods
  - serviceaccounts
  verbs:
  - get
  - list
...
```

Reply:
Done, thanks!